src/features/form/containers/GroupContainer.tsx

```diff
@@ -189,6 +191,8 @@ export function GroupContainer({ id, container, components }: IGroupProps): JSX.
     return null;
   }
 
+  const isNested = typeof container.baseComponentId === 'string';
```

Comment:
Adding type for strict typechecking.
```suggestion
  const isNested: boolean = typeof container.baseComponentId === 'string';
```

Reply:
Personally, I'm not a fan of "strict" typechecking. TypeScript is best when it's not in your way, and TypeScript is smart enough to figure out that the return value from a comparison is a boolean. There's no need to write anything more than we absolutely _have_ to. In the future I want to enable `noImplicitAny` in `tsconfig.json` so that we are nagged to specify types wherever TypeScript _can't_ automatically infer the type.

Reply:
I must have misunderstood some previous talk about typechecking then 😅 Thanks for clarifying! I have no problem personally with keeping the types inferred where they can be 👍

Reply:
This is not code that this PR touches, so this is out of scope.